pkg/threescale/threescale.go

```diff
@@ -115,68 +117,95 @@ func (s *Threescale) extractProxyConf(cfg *config.Params, c *sysC.ThreeScaleClie
 	return pce, proxyConfErr
 }
 
-// isAuthorized returns code 0 if authorization is successful
-// based on grpc return codes https://github.com/grpc/grpc-go/blob/master/codes/codes.go
-func (s *Threescale) isAuthorized(cfg *config.Params, request authorization.InstanceMsg, proxyConf sysC.ProxyConfigElement, client *backendC.ThreeScaleClient) (rpc.Status, error) {
+// isAuthorized - is responsible for parsing the incoming request and determining if it is valid, building out the request to be sent
+// to 3scale and parsing the response. Returns code 0 if authorization is successful based on
+// grpc return codes https://github.com/grpc/grpc-go/blob/master/codes/codes.go
+func (s *Threescale) isAuthorized(svcID string, request authorization.InstanceMsg, proxyConf sysC.ProxyConfig, client *backendC.ThreeScaleClient) rpc.Status {
+	var (
+		// Application ID authentication pattern - App Key is optional when using this authn
+		appID, appKey string
+
+		// Application Key auth pattern
+		userKey string
+
+		// Function to be called when authn patter has been determined
+		authRep authRepFn
+	)
+
 	if request.Action.Path == "" {
-		return status.WithInvalidArgument("missing request path"), nil
+		request.Action.Path = "/"
```

Comment:
as stated in the removal of the test, I think erroring out is best unless we can be sure this won't ever happen.

Reply:
See comment above. I have no problem reverting this if you think its best.

Reply:
Resolved I believe?